hokusai/staging.yml

```diff
@@ -210,7 +218,7 @@ spec:
             periodSeconds: 30
           resources:
             requests:
-              cpu: 100m
+              cpu: 50m
```

Comment:
[near-zero usage](https://app.datadoghq.com/dashboard/2n5-6tr-ypp/kubernetes-cpumemory-usage-by-deployment?from_ts=1616080309689&live=true&tile_size=s&to_ts=1616685109689&tpl_var_deployment=convection-sidekiq&tpl_var_env=staging&tpl_var_hpa=convection-sidekiq) in staging.